CONTRIBUTING.md

```diff
@@ -58,33 +58,38 @@ Each new pull request should:
 
 ## Creating a new release
 
-### Minor/patch releases
-Minor and patch releases only need to change this repo. The Helm chart and deploy scripts
+### Patch releases
+Patch releases only need to change this repo. The Helm chart and deploy scripts
 will automatically pull in the latest changes.
 
-To deploy a minor or patch release, follow steps 2 and 3 from "Major releases" below.
+If the release involves changes to anything in the `deploy/` folder, it needs to be a minor
+or major release in order to prevent breaking the Helm chart.
 
-### Major releases
-Major releases need to change both this repository and the
+To deploy a patch release, follow steps 2 and 3 below.
+
+### Minor/Major releases
+Minor and major releases need to change both this repository and the
 [Helm chart repo](https://github.com/reactiveops/charts/).
 
 The steps are:
-1. Create a PR in the [charts repo](https://github.com/reactiveops/charts/)
-    1. Use a branch named `polaris-latest`
-    2. Bump the version number in:
+1. Modify the [Helm chart](https://github.com/reactiveops/charts/stable/polaris)
+    1. Bump the version number in:
         1. stable/polaris/README.md
         2. stable/polaris/Chart.yaml
         3. stable/polaris/values.yaml
-    3. **Don't merge yet!**
+    2. Make any additional changes necessary (e.g. new RBAC permissions)
```

Comment:
This sentence is a little vague - is it specifically calling out the `./deploy` manifests that may need to be changed?